low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -56,8 +57,9 @@ class SSPLDevDeploy:
     def __init__(self, args):
         """Initialize required config variables."""
         self.args = args
+        # TODO: replace this by main branch URL
         # self.CORTX_MONITOR_BASE_URL = "https://raw.githubusercontent.com/Seagate/cortx-monitor/main"
-        self.CORTX_MONITOR_BASE_URL = "https://raw.githubusercontent.com/Seagate/cortx-monitor/1cec21efc4f5776cabc6e749ed4a57fa90466972"
+        self.CORTX_MONITOR_BASE_URL = "https://raw.githubusercontent.com/satish-seagate/cortx-monitor/eos-16520-main-converting_sspl_dev_deploy_shell_to_python"
```

Comment:
change later again to main once merged

Reply:
Sure.

Reply:
@satish-seagate, This has to be take care in final commit.

Reply:
after commit right!?

Reply:
No. If all good with private repo, then the final commit is to change this alone before merge.

Reply:
Done